test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/IncludeSearchTests.cs

```diff
@@ -490,6 +490,25 @@ public async Task GivenARevIncludeSearchExpressionWithMultipleIncludes_WhenSearc
             ValidateBundleUrl(Client.HttpClient.BaseAddress, ResourceType.Patient, query, bundle.Link[0].Url);
         }
 
+        [InlineData("_include")]
+        [InlineData("_revinclude")]
+        [Theory]
+        public async Task GivenAnIncludeSearchExpressionWithLocationLinkedToItself_WhenSearched_ThenCorrectBundleShouldBeReturned(string includeType)
```

Comment:
Would a test showing that this also works for cyclical references have value? IE: resource 1 references resource 2 and resource 2 references resource 1.

Reply:
Good idea, I think this can only be done with SQL (:iterate)